aws_lambda_powertools/utilities/parameters/ssm.py

```diff
@@ -87,15 +87,15 @@ def __init__(self, config: Optional[Config] = None, boto3_session: Optional[boto
 
         super().__init__()
 
-    def get(
+    def get(  # type: ignore[override]
```

Comment:
would be great to have a comment on the need for override so anyone can understand when coming back to the code.

Reply:
Ok!

Reply:
Added this comments:
```
    # We break Liskov substitution principle due to differences between signature of this method and corresponding method in superclass.
    # We ignore mypy error, as change to the signature is a breaking change to users.
```